runner/run_benchmark.py

```diff
@@ -136,7 +137,7 @@ def check_result_files(run_name: str, agent_name: str, force_removal: bool = Fal
                 ):
                     colour_print("red", "Run aborted.")
                     return
-                for file in result_files:
+                for file in result_files + [make_runstats_path(run_name, agent_name), make_master_log_path(run_name, agent_name)]:
```

Comment:
Not sure if those files are present at all times. You might need to check before calling `remove`.